packages/vscode/src/util.ts

```diff
@@ -12,11 +12,29 @@ import {
   denyListDarkColorThemes,
   denyListLightColorThemes,
 } from './denyListColorThemes'
+import { homedir } from 'os'
+import { readdirSync } from 'fs'
+import path from 'path'
 
 export function isDebugOrTestSession(): boolean {
   return env.sessionId === 'someValue.sessionId'
 }
 
+export function checkForOtherPrismaExtension(extensionId: string) {
+  const files = readdirSync(
+    path.join(homedir(), '.vscode/extensions'),
+  ).filter((fn) => fn.startsWith('prisma.prisma-'))
```

Comment:
Can this (`prisma.prisma-`) be (`Prisma.prisma-`) in some cases? I have seen a capitalized version of Prisma somewhere (I think in the market place). 

If you are sure, then it is ok.
If not, then you can maybe add a lowercase before comparison.

Reply:
In the folder I have only seen lowercase versions, but I'm not 100% if that is always the case. Adding the toLowerCase just in case